x/dispensation/keeper/distrubtions_test.go

```diff
@@ -0,0 +1,24 @@
+package keeper_test
+
+import (
+	"github.com/Sifchain/sifnode/x/dispensation/test"
+	"github.com/Sifchain/sifnode/x/dispensation/types"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestKeeper_GetDistributions(t *testing.T) {
+	app, ctx := test.CreateTestApp(false)
+	keeper := app.DispensationKeeper
+	for i := 0; i < 10; i++ {
+		name := uuid.New().String()
+		distribution := types.NewDistribution(types.Airdrop, name)
+		err := keeper.SetDistribution(ctx, distribution)
+		assert.NoError(t, err)
+		_, err = keeper.GetDistribution(ctx, name, types.Airdrop)
```

Comment:
Are there other assertion to put on distribution res?

Reply:
I added the test for GetDistributions and it has a check for the total list .